dataprofiler/data_readers/filepath_or_buffer.py

```diff
@@ -0,0 +1,31 @@
+from io import open, StringIO, BytesIO
+
+from . import data_utils
+
+class FileOrBufferHandler():
+    def __init__(self, stream_or_buffer, open_method='r', options=dict()):
```

Comment:
instead of `stream_or_buffer`, lets stick closer to convention and use `filepath_or_buffer`